packages/contracts/contracts/zkopru/controllers/Coordinatable.sol

```diff
@@ -59,6 +59,10 @@ contract Coordinatable is Storage {
         delete Storage.chain.proposers[proposerAddr];
     }
 
+    function coordinatorExitBlock(address coordinator) public view returns (uint) {
```

Comment:
How about move this function to Reader.sol? 
https://github.com/zkopru-network/zkopru/blob/develop/packages/contracts/contracts/zkopru/storage/Reader.sol

Reply:
This is a bit of a hack. In the BurnAuction the round should be opened to anyone if no proposal is made in the first half of the round. This requires knowledge of the timestamp of the latest proposal. The `coordinatorExitBlock` lets me approximate whether or not the latest proposal was made in the current round but it assumes 15 second block times and potentially opens a round that shouldn't be open where proposals were only made early in the round (e.g. one proposal in the first minute and no others).

Is this margin of error okay? i.e. punish coordinators that don't use their time. Or should we store a `uint256 latestProposalTimestamp` or similar?

Reply:
Disregard, I refactored the auction system to use blocks instead of seconds. This works with the existing variables in zkopru.